core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-language-selector/edit-ema-language-selector.component.ts

```diff
@@ -0,0 +1,59 @@
+import { CommonModule } from '@angular/common';
+import { Component, EventEmitter, Input, OnInit, Output, inject } from '@angular/core';
+
+import { ButtonModule } from 'primeng/button';
+import { ListboxModule } from 'primeng/listbox';
+import { OverlayPanelModule } from 'primeng/overlaypanel';
+
+import { DotLanguagesService } from '@dotcms/data-access';
+import { DotLanguage } from '@dotcms/dotcms-models';
+
+interface DotLanguageWithLabel extends DotLanguage {
+    label: string;
+}
+
+@Component({
+    selector: 'dot-edit-ema-language-selector',
+    standalone: true,
+    imports: [CommonModule, OverlayPanelModule, ListboxModule, ButtonModule],
+    templateUrl: './edit-ema-language-selector.component.html',
+    styleUrls: ['./edit-ema-language-selector.component.scss']
+})
+export class EmaLanguageSelectorComponent implements OnInit {
+    @Output() languageSelected: EventEmitter<number> = new EventEmitter();
```

Comment:
```suggestion
    @Output() selected: EventEmitter<number> = new EventEmitter();
```

I already a language selector so is redundant 😂